service.py

```diff
@@ -51,6 +51,16 @@ def showErrorNotification(message):
                                   xbmcgui.NOTIFICATION_ERROR, 5000)
 
 
+def get_user_input():
```

Comment:
just an idea but maybe name it something like `get_local_user_input()` . A actual send url is also somewhat of a user input.